config/m4/verbs.m4

```diff
@@ -27,7 +27,12 @@ AC_DEFUN([CHECK_VERBS_ATTRIBUTE], [
     ])
 ])
 
-
+# Check attributes
+# Usage: CHECK_VERBS_MEMBER([attribute], [header file], [definition])
+#
+AC_DEFUN([CHECK_VERBS_MEMBER], [
+    AC_CHECK_MEMBER( $1, [AC_DEFINE_UNQUOTED([DEFINED_$3], [1], [Define to 1 if attribute $1 is supported])], [], [#include <$2>])
+])
```

Comment:
Probably you need to use square brackets to close header file (see: https://www.gnu.org/software/autoconf/manual/autoconf-2.64/html_node/Generic-Structures.html)
```
AC_DEFUN([CHECK_VERBS_MEMBER], [
    AC_CHECK_MEMBER( $1, [AC_DEFINE_UNQUOTED([DEFINED_$3], [1], [Define to 1 if attribute $1 is supported])], [], [[#include <$2>]])
])```